errors.go

```diff
@@ -18,4 +18,7 @@ var (
 	// that server does not allow subscribing to the same channel twice for
 	// the same connection.
 	ErrDuplicateSubscription = errors.New("duplicate subscription")
+	// ErrSubscribeClosed returned if subscription was closed
+	// for subscribe to this channel should use Client's NewSubscription
+	ErrSubscribeClosed = errors.New("subscribe is closed")
```

Comment:
`ErrSubscriptionClosed` is better (`Close` means that Subscription is Closed not only for Subscribe but for all operations actually - it's not usable anymore).